lib/xnvme_be_linux_async_ucmd.c

```diff
@@ -183,6 +173,8 @@ xnvme_be_linux_ucmd_io(struct xnvme_cmd_ctx *ctx, void *dbuf, size_t dbuf_nbytes
 	ctx->cmd.common.mptr = (uint64_t)mbuf;
 	ctx->cmd.common.dptr.lnx_ioctl.metadata_len = mbuf_nbytes;
 
+	memcpy(&sqe->addr3, &ctx->cmd.common, NVME_URING_CMD_SIZE);
```

Comment:
To avoid copying more than the NVMe-command, how about limiting this to ``64`` instead of ``NVME_URING_CMD_SIZE``? And possibly adding a XNVME_STATIC_ASSERT in the internal header verifying that NVME_URING_CMD_SIZE is >= 64?